### ISSUE TITLE
[Bug] SSL: CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by rabea98** [notcammy/PySnapStories#1]
So the script stopped working, looks like the api broke.
Here's my log from the script:

```
D:\PySnapStories-master>python pysnapstories.py elsiehewitt
-----------------------------------------------------------------------------------------------
PYSNAPSTORIES (SCRIPT V2.1 - PYTHON V3.7.4) - 04:21:25 AM
-----------------------------------------------------------------------------------------------
Treating input as username. (no ID was detected)
Starting download for user: elsiehewitt
Something went wrong: HTTPSConnectionPool(host='storysharing.snapchat.com', port=443): Max retries exceeded with url: /v1/fetch/elsiehewitt (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1076)')))
The script cannot continue, exiting.

D:\PySnapStories-master>python pysnapstories.py sommerray
-----------------------------------------------------------------------------------------------
PYSNAPSTORIES (SCRIPT V2.1 - PYTHON V3.7.4) - 04:21:26 AM
-----------------------------------------------------------------------------------------------
Treating input as username. (no ID was detected)
Starting download for user: sommerray
Something went wrong: HTTPSConnectionPool(host='storysharing.snapchat.com', port=443): Max retries exceeded with url: /v1/fetch/sommerray (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1076)')))
The script cannot continue, exiting.```

**Comment by dvingerh** [comment]
Fixed in 75f35a52574473045854667198b9a3dd8d19164c

**Comment by rabea98** [comment]
Thanks for the quick fix and for maintaining the script.
